### PR TITLE
MSTR-39: Add crm integration api endpoints

### PR DESCRIPTION
**Pull request by ConR24**

### CODE REVIEW

File: src/js/lib/jquery.kazoosdk.js [hunk 1]
```diff
@@ -525,6 +525,12 @@
 				'deleteNotification': { verb: 'DELETE', url: 'accounts/{accountId}/notifications/{notificationId}' },
 				'getDnsEntries': { verb: 'GET', url: 'accounts/{accountId}/whitelabel/domains' },
 				'checkDnsEntries': { verb: 'POST', url: 'accounts/{accountId}/whitelabel/domains?domain={domain}' }
+			},
+			crm: {
+				'getIntegration': { verb: 'GET', url: 'accounts/{accountId}/crm_integration'},
+				'setIntegration': { verb: 'PUT', url: 'accounts/{accountId}/crm_integration'},
```
Comment (azefiel) [inline src/js/lib/jquery.kazoosdk.js]: Is the intent here to create an integration or does set has a different meaning? If it is, we should use `create` to keep that action uniform throughout the sdk.
```suggestion
				'createIntegration': { verb: 'PUT', url: 'accounts/{accountId}/crm_integration'},
```

File: src/js/lib/jquery.kazoosdk.js [hunk 2]
```diff
@@ -525,6 +525,12 @@
...
+				'getTriggers': { verb: 'GET', url: 'accounts/{accountId}/crm_trigger'},
```
Comment (azefiel) [inline src/js/lib/jquery.kazoosdk.js]:
```suggestion
				'getTrigger': { verb: 'GET', url: 'accounts/{accountId}/crm_trigger'},
```

File: src/js/lib/jquery.kazoosdk.js [hunk 3]
```diff
@@ -525,6 +525,12 @@
...
+				'setTriggers': { verb: 'PUT', url: 'accounts/{accountId}/crm_trigger'}
```
Comment (azefiel) [inline src/js/lib/jquery.kazoosdk.js]: Same here:
```suggestion
				'createTrigger': { verb: 'PUT', url: 'accounts/{accountId}/crm_trigger'}
```

File: src/js/lib/jquery.kazoosdk.js (hunk 1 above)
Comment (ConR24) [reply src/js/lib/jquery.kazoosdk.js]: The intent is to create a new integration, I'll fix the name.

**Pull request merged**